### PR TITLE
Introduce more components

### PR DESCRIPTION
**Pull request by dimitrisnl**

### CODE REVIEW

**Comment by dimitrisnl** [comment]
So far

File: lib/App.jsx [hunk 1]
```diff
@@ -25,6 +25,9 @@ import {
 
 import './global.css'
 
+import 'typeface-nunito'
+import "circular-std";
```
Comment (jviotti) [inline lib/App.jsx]: Can we change to single quotes and avoid the semi-colon here? I'm surprised the linter didn't catch this

File: lib/components/users.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  Txt,
+  Divider,
+} from 'rendition';
+
+export const name = 'Users';
+
+export const variants = metadata => {
+  const combinations = [];
+
+  if (metadata.data.github.usedBy) {
```
Comment (jviotti) [inline lib/components/users.jsx]: Maybe the check should be that the project is used by at least 1 project?

File: lib/components/users.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+  return combinations;
+};
+
+const GITHUB_PROFILE_PATH = `https://github.com`;
+
+export const render = props => {
+  const list = props.users.map(({ repo, owner, description }) => {
+    return (
+      <>
```
Comment (jviotti) [inline lib/components/users.jsx]: Is this a typo?

File: theme/tmp/dev-server/index.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en" data-reactroot=""><head><title>landr</title><meta name="description" content="source code = website 🏠"/><meta charSet="UTF-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link href="https://fonts.googleapis.com/css?family=Nunito&amp;display=swap" rel="stylesheet"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/solid.css" integrity="sha384-ioUrHig76ITq4aEJ67dHzTvqjsAP/7IzgwE7lgJcg2r7BRNGYSK0LwSmROzYtgzs" crossOrigin="anonymous"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/brands.css" integrity="sha384-i2PyM6FMpVnxjRPi0KW/xIS7hkeSznkllv+Hx/MtYDaHA5VcF0yL3KVlvzp8bWjQ" crossOrigin="anonymous"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/fontawesome.css" integrity="sha384-sri+NftO+0hcisDKgr287Y/1LVnInHJ1l+XC7+FOabmTTIK0HnE2ID+xxvJ21c5J" crossOrigin="anonymous"/></head><body><div id="root"></div></body></html>
```
Comment (jviotti) [inline theme/tmp/dev-server/index.html]: What is this theme directory?

File: lib/components/users.jsx (hunk 3 above)
Comment (dimitrisnl) [reply lib/components/users.jsx]: No @jviotti 	This is a fragment https://reactjs.org/docs/fragments.html

Needed when you want to render two adjacent components without wrapping them with unnecessary dom elements

File: theme/tmp/dev-server/index.html (hunk 4 above)
Comment (dimitrisnl) [reply theme/tmp/dev-server/index.html]: Probably an untracked file I had from the previous implementation - removing

**Pull request merged**